app/modules/cdr/endpoints_cdr.py

```diff
@@ -351,6 +355,197 @@ async def get_online_sellers(
     return await cruds_cdr.get_online_sellers(db)
 
 
+def construct_dataframe_from_users_purchases(
```

Comment:
Move the util to `app/modules/cdr/utils_cdr.py`

Reply:
Done